docs/src/pages/components/masonry/BasicMasonry.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import * as React from 'react';
+import Masonry from '@material-ui/lab/Masonry';
+import MasonryItem from '@material-ui/lab/MasonryItem';
+import Box from '@material-ui/core/Box';
+
+export default function BasicMasonry() {
```

Comment:
I would maybe suggest using the `ImageMasonary` as a first demo, and then have a demo with description like: "you can also use the Masonry with `div` elements", and add this demo. The main reason is that the demo with the images is more likely closer to how someone may use the component.